doc/classes/AudioEffectChorus.xml

```diff
@@ -14,78 +14,90 @@
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the cutoff frequency of a specific voice.
 			</description>
 		</method>
 		<method name="get_voice_delay_ms" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the delay of a specific voice in milliseconds.
 			</description>
 		</method>
 		<method name="get_voice_depth_ms" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the depth of a specific voice filter in milliseconds.
 			</description>
 		</method>
 		<method name="get_voice_level_db" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the volume of a specific voice in decibels.
 			</description>
 		</method>
 		<method name="get_voice_pan" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the pan level of a specific voice.
 			</description>
 		</method>
 		<method name="get_voice_rate_hz" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the filter rate frequency of a specific voice.
 			</description>
 		</method>
 		<method name="set_voice_cutoff_hz">
 			<return type="void" />
 			<param index="0" name="voice_idx" type="int" />
 			<param index="1" name="cutoff_hz" type="float" />
 			<description>
+				Sets the cutoff frequency of a specific voice in hertz (number of samples per second). Cutoff frequency refers to the top-most frequency beyond which the signal is blocked. This can be used to create a low pass filter to simulate a "muffled" sound. The recommended range is [code]0[/code] to [code]20500[/code].
 			</description>
 		</method>
 		<method name="set_voice_delay_ms">
 			<return type="void" />
 			<param index="0" name="voice_idx" type="int" />
 			<param index="1" name="delay_ms" type="float" />
 			<description>
+				Sets the delay of a specific voice in milliseconds. Longer delay creates more audible separation between the voices. The recommended range is [code]0[/code] to [code]50[/code]. [b]Note: [/b] the maximum recommended value can be exceeded to create longer more noticeable separation.
```

Comment:
A note should be on a new line, but even then, I believe this note is redundant and not necessary.
```suggestion
				Sets the delay of a specific voice in milliseconds. Longer delay creates more audible separation between the voices. Values between [code]0.0[/code] and [code]50.0[/code] are recommended.
```
In hindsight, one could argue the recommended range does not need to be specified, either. Editing the property in the Inspector would make it clear, but this is a method, not a property...

Reply:
I suppose the word "recommended" is already acknowledging that the range can be exceeded, if one wants to experiment, so the note might be redundant. I guess I was encouraging experimentation, and the class reference isn't really the right place for that...